.travis.yml

```diff
@@ -22,7 +22,7 @@ notifications:
     on_failure: change
 
 node_js:
-  - "0.10"
+  - "4.5"
```

Comment:
@vladikoff curious if you have an idea what could be going on with failing tests. The server mock tests pass locally for me but fail on travis when using node 0.10. When upgrading to 4.5, they pass. Both server mock and local tests fail with either 0.10 or 4.5. 

Travis logs seem to point to auth-mailer build errors? https://travis-ci.org/mozilla/fxa-js-client/jobs/160703028#L704